laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
@GrahamCampbell the issue is that this advisory database will reference the security issue, but it will not document it. There will be no explanation about the issue (this is precisely what the link given in the advisory is meant to cover)